requirements.txt

```diff
@@ -40,3 +40,4 @@ sentence-transformers==2.0.0
 spacy==3.0.7
 spacy-transformers==1.0.3
 torch==1.9.0
+elasticsearch==8.3.3
```

Comment:
Maybe you can also update the `setup.py`. However, I am not sure what is the best place to put that package. It is going to depend on how we handle this in the future (if we keep this into this repo and where, if we want `elasticsearch` directly in the code base or just in notebooks).

Reply:
elasticsearch and decouple added to requirements and setup

Reply:
Sorry, do you mind putting them in alphabetical order in the `setup.py` ?

Reply:
done

Reply:
Perfect! Thank you